config/develop/namespaced/s3-to-glue-lambda.yaml

```diff
@@ -4,11 +4,14 @@ template:
   artifact_bucket_name: {{ stack_group_config.template_bucket_name }}
   artifact_prefix: '{{ stack_group_config.namespace }}/src/lambda'
 dependencies:
-  - develop/s3-to-glue-lambda-role.yaml
+  - develop/namespaced/s3-to-glue-lambda-role.yaml
   - develop/namespaced/glue-workflow.yaml
 stack_name: '{{ stack_group_config.namespace }}-lambda-S3ToGlue'
 stack_tags: {{ stack_group_config.default_stack_tags }}
 parameters:
-  S3SourceBucketName: {{ stack_group_config.input_bucket_name }}
-  S3ToGlueRoleArn: !stack_output_external s3-to-glue-lambda-role::RoleArn
+  SQSQueueUrl: !stack_output_external "{{ stack_group_config.namespace }}-sqs-S3ToLambda::PrimaryQueueUrl"
+  SQSQueueArn: !stack_output_external "{{ stack_group_config.namespace }}-sqs-S3ToLambda::PrimaryQueueArn"
+  S3ToGlueRoleArn: !stack_output_external "{{ stack_group_config.namespace }}-s3-to-glue-lambda-role::RoleArn"
   PrimaryWorkflowName: !stack_output_external "{{ stack_group_config.namespace }}-glue-workflow::WorkflowName"
+  LambdaBatchSize: '10'
```

Comment:
Can we push these docs: https://docs.google.com/document/d/1K36ViBjzh32W32rA6mFzXEehyO2ee5qqAg2k7MHIF3g/edit#heading=h.urvxhb9crjdz to https://sagebionetworks.jira.com/wiki/spaces/IBC/pages/2733867066/Design+Documents